### PR TITLE
Deploy linked binaries if available + some cleanup

### PR DESCRIPTION
**Pull request by adrianstevens**

### CODE REVIEW

**Review by ctacke — APPROVED** [review]

File: Source/v2/Meadow.Cli/AppManager.cs [hunk 1]
```diff
@@ -36,27 +37,52 @@ private static bool IsXmlDoc(string file)
...
-
         logger?.LogInformation("Generating list of files to deploy...");
+
         foreach (var file in dependencies)
         {
             // TODO: add any other filtering capability here
-
-            if (!includePdbs && IsPdb(file)) continue;
-            if (!includeXmlDocs && IsXmlDoc(file)) continue;
+            if (!includePdbs && IsPdb(file)) { continue; }
```
Comment (stevenkuhn) [inline Source/v2/Meadow.Cli/AppManager.cs:84]: Love having the inline `{ }` here. :)

File: Source/v2/Meadow.Package/PackageManager.cs [hunk 2]
```diff
@@ -230,10 +228,10 @@ private void CreateEntry(ZipArchive archive, string fromFile, string entryPath)
 
     public static FileInfo[] GetAvailableBuiltConfigurations(string rootFolder, string appName = "App.dll")
     {
-        // check if we were give path to a project file, not the folder of the project file.
+        // check if we were give path to a project file, not the folder of the project file
         if (File.Exists(rootFolder))
         {
-            rootFolder = Path.GetDirectoryName(rootFolder) ?? ""; // extreact the folder name or if invalid, use the current directory.
+            rootFolder = Path.GetDirectoryName(rootFolder) ?? ""; // extreact the folder name or if invalid, use the current directory
```
Comment (stevenkuhn) [inline Source/v2/Meadow.Package/PackageManager.cs:234]: Small nit on "extreact"

File: Source/v2/Meadow.Cli/AppManager.cs [hunk 3]
```diff
@@ -14,12 +15,12 @@ private static bool MatchingDllExists(string file)
 
     private static bool IsPdb(string file)
     {
-        return String.Compare(Path.GetExtension(file), ".pdb", StringComparison.OrdinalIgnoreCase) == 0;
+        return string.Compare(Path.GetExtension(file), ".pdb", StringComparison.OrdinalIgnoreCase) == 0;
```
[the remark this answers is not in the document]
Comment (adrianstevens) [reply Source/v2/Meadow.Cli/AppManager.cs:18]: yeah the CLI is a .NET 8 app so I get the fancy APIs ;)

File: Source/v2/Meadow.Cli/AppManager.cs (hunk 1 above)
Comment (adrianstevens) [reply Source/v2/Meadow.Cli/AppManager.cs:84]: hahaha

File: Source/v2/Meadow.Package/PackageManager.cs (hunk 2 above)
Comment (adrianstevens) [reply Source/v2/Meadow.Package/PackageManager.cs:234]: nice - I'll fix

**Pull request merged**